wealth-management-client/src/app/reports/recommendations/recommendation-grid.component.html

```diff
@@ -1,10 +1,12 @@
 <p-dataTable [value]="stockRecommendationList" [sortMode]="multiple">
-  <p-column field="script" header="Script" sortable="true"></p-column>
-  <p-column field="name" header="Name" sortable="true"></p-column>
+  <p-column field="script" header="Script" sortable="true" filter="true"></p-column>
+  <!-- p-column field="name" header="Name" sortable="true"></p-column -->
   <p-column field="rcomPrice" header="Recommended Price" sortable="true"></p-column>
-  <p-column field="buyPrice" header="Buy Price" sortable="true"></p-column>
+  <!-- p-column field="buyPrice" header="Buy Price" sortable="true"></p-column -->
   <p-column field="targetPrice" header="Recommended Price" sortable="true"></p-column>
+  <p-column field="rtargetPercentage" header="Recommended Target" sortable="true"></p-column>
+  <p-column field="days" header="Number of Days" sortable="true"></p-column>
```

Comment:
Sort is not working...

Reply:
Sort was working for me. Is it not working after adding filter?